pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/NamespaceStats.java

```diff
@@ -61,7 +62,8 @@
         }
     }
 
-    public NamespaceStats() {
+    public NamespaceStats(int ratePeriodInSeconds) {
+        this.ratePeriodInSeconds  = ratePeriodInSeconds;
```

Comment:
What about adding a precondition in order to prevent bad errors like division by zero later?